### ISSUE TITLE
File metadata get stuck in Validating when data files are corrupted

### ISSUE THREAD

**Issue by aaclan-ebi** [HumanCellAtlas/ingest-central#448]
https://humancellatlas.slack.com/archives/C4LTU5RT8/p1562064099043500

This happens when a corrupted file fails to be opened by the fastq-validator and a non-expected error string format is returned

Acceptance criteria:

* The fastq-validator emits any and all error messages in an expected format
* The ingest-validator marks files invalid if it cannot parse the error string

**Comment by justincc** [comment]
"In ingest we need to timeout on upload non-response and show the user why this has failed, possibly with a manual retry button. I'm assuming we are already doing automatic retries but if not we'll need to do those too." - My opinion

**Comment by justincc** [comment]
@parthshahva is aware and we can work with him on this when we schedule this ticket.

**Comment by justincc** [comment]
@alegria We looked at this in the task estimation but it wasn't easy to figure out exactly what the problem is. Could you summarize the thread here? It sounds like this will need more investigation before we can estimate?

**Comment by aaclan-ebi** [comment]
Steps To Repro:
1. Create a submission
2. Create file metadata
3. Upload a corrupted fastqzip file (any invalid fastq file, could ask Enrique for a sample

Actual behavior:
File is stuck in Validating

Expected behavior:
File should be marked as invalid.

**Comment by prabh-t** [comment]
Currently, validation state will remain validating if we are not able to parse the file validation result from upload, and will only change back to draft when a new file is uploaded again. As mentioned in the previous comment, we should maybe change state to invalid if unable to parse response from upload srv.

**Comment by prabh-t** [comment]
Also there is a retry mechanism in place -- validator would normally make 5 attempts to external requests (to core, upload, ontology service) in case of errors. But no timeout as far as I can tell.

**Comment by justincc** [comment]
Yes, we need timeouts on this stuff.
